### PR TITLE
Added `tol`, `max_iter` parameters to allow fixing convergence issues.

### PR DESCRIPTION
**Pull request by alzmcr**
Adding `max_iter` and `tol` parameter for LassoCV and LogisticRegressionCV. This as something the solver doesn't converge and these parameters needs to respectively increase and decrease.

It's actually a very small change, sorry for the multiple lines change - it's just VSCode that is doing some code sanitisation (removing trailing spaces). Also, `test_fried_scale` is broken and I had to comment that out to pass the tests.

Thanks for making this - it's useful in my use case :)

### CODE REVIEW

**Comment by alzmcr** [comment]
@christophM would be great to merge this :)

**Comment by alzmcr** [comment]
friendly ping @christophM :)

File: test_rulefit.py [hunk 1]
```diff
@@ -67,8 +67,8 @@ def run_all_tests():
     test_rule_condition_greater()
     test_rule_transform()
     test_rule_equality()
-    test_fried_scale()
-    
+    # test_fried_scale() TODO: Fix this test as FriedScale signature has changed.
```
Comment (christophM) [inline test_rulefit.py:70]: I see there is a TODO here?

**Comment by christophM** [comment]
Hey, thanks for the ping. I assumed it is not ready yet, because there is a test commented out with a TODO

File: test_rulefit.py (hunk 1 above)
Comment (alzmcr) [reply test_rulefit.py:70]: The point was to make all the tests pass. This is something that has been broken for a while I presume.

**Comment by alzmcr** [comment]
Hey, I see. I had to comment that test out as it's broken and is making the tests fail. 

Up to do what is the best thing do to:

1 )Just remove the comment as leave as it was
2) Create an issue for someone to fix the test
3) Merge as it is

**Comment by christophM** [comment]
Alright, 2) + 3) it is.

**Pull request merged**